pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -166,7 +183,7 @@ func TestRunScorePlugins(t *testing.T) {
 		name        string
 		registry    Registry
 		plugins     *config.Plugins
-		injectedRes map[string]injectedResult
+		injectedRes []config.PluginConfig
```

Comment:
nit: pluginConfigs

Reply:
fixed in https://github.com/kubernetes/kubernetes/pull/82483/commits/b3a82e3b93af38c478b6e65145e24e4730ef7882